tox.ini

```diff
@@ -1,16 +1,23 @@
 [tox]
-envlist = lint, test
+envlist = flake8, pylint, test
 skipsdist=True
 
 [testenv]
 ignore_errors = False
 basepython = python3.6
 
-[testenv:lint]
+[testenv:flake8]
 use_develop = True
 commands = flake8 --max-line-length 99 --max-complexity 12 nim
 deps = flake8
 
+[testenv:pylint]
+use_develop = True
+# Ignore return code of Pylint for now to not make the build fail. Remove the
+# dash once we've fixed all the issues.
+commands = - pylint nim
```

Comment:
What's the `-` for?

Reply:
See comment above that line :wink:

Reply:
Well then, I guess that answers both of my questions! :)